src/components/reference/Reference.js

```diff
@@ -151,10 +163,18 @@ export default class Reference extends React.Component {
                                     <Icon className="material-icons">close</Icon>
                                 </IconButton>
                                 <IconButton
+                                    title="Open reference page &#013;(in a new tab)"
                                     color="default"
                                     style={newTabStyle}
                                     onClick={this.handleOpen}>
-                                    <Icon className="material-icons">open_in_new</Icon>
+                                    <Icon className="material-icons">menu_book</Icon>
+                                </IconButton>
+                                <IconButton
+                                    title="Open asset reference page &#013;(in a new tab)"
+                                    color="default"
+                                    style={assetReferenceBtn}
+                                    onClick={this.assetHandleOpen}>
+                                    <Icon className="material-icons-outlined">cloud_download</Icon>
```

Comment:
I feel like cloud_download is not a best icon for representing asset page. "settings_system_daydream" might be more appropriate?